hawkbit-sdk/hawkbit-sdk-dmf/src/main/java/org/eclipse/hawkbit/sdk/dmf/amqp/Amqp.java

```diff
@@ -38,20 +39,31 @@ public void stop() {
         vHosts.values().forEach(VHost::stop);
     }
 
-    public VHost getVhost(final DMF dmf) {
+    public VHost getVhost(final DMF dmf, final boolean isEnvLocal) {
```

Comment:
since this boolean parameter is interpreted as "create queues" better rename it to createQueues (or initVHost ?)